bin/sandbox

```diff
@@ -51,6 +51,8 @@ cd ./sandbox
 cat <<RUBY >> Gemfile
 gem 'solidus', github: 'solidusio/solidus', branch: '$BRANCH'
 gem 'solidus_auth_devise', '>= 2.1.0'
+gem 'omniauth-bolt', github: 'nebulab/omniauth-bolt'
```

Comment:
you are adding the gem here but not in the dependencies - could be the reason why the tests are failing